### PR TITLE
Add user/books models relation

### PR DESCRIPTION
**Pull request by ese-varo**
Create one to many relation between user and book

### CODE REVIEW

File: src/routes/book.js [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
-  router.get("/", books.findAll);
+  router.get("/user/:id", [authJwt.verifyToken], books.findAll);
 
   // Retrieve a single Book with id
-  router.get("/:id", books.findOne);
+  router.get("/:id", [authJwt.verifyToken], books.findOne);
 
   // Update a Book with id
-  router.put("/:id", books.update);
+  router.put("/:id", [authJwt.verifyToken], books.update);
```
Comment (OmauryMarmolejo) [inline src/routes/book.js]: Do you need the `[]` wraper?

File: src/controllers/book.js [hunk 2]
```diff
@@ -40,10 +41,9 @@ exports.create = (req, res) => {
...
 
 exports.findAll = (req, res) => {
-  const title = req.query.title;
-  var condition = title ? { title: { [Op.iLike]: `%${title}%` } } : null;
+  const userId = req.params.id;
 
-  Book.findAll({ where: condition })
+  Book.findAll({ where: { userId: userId } })
     .then(data => {
       res.send(data);
```
Comment (OmauryMarmolejo) [inline src/controllers/book.js]: https://www.loginradius.com/blog/async/callback-vs-promises-vs-async-await/

File: server.js [hunk 3]
```diff
@@ -13,6 +13,10 @@ var corsOptions = {
 const db = require("./src/models");
 
 db.sequelize.sync();
+// on development mode
+// db.sequelize.sync({ force: true }).then(() => {
+//   console.log("Drop and re-sync db.");
+// });
```
Comment (LoginALee) [inline server.js]: You could delete these lines

**Review by LoginALee — APPROVED** [review]
Great job Alvaro :rocket:

File: src/routes/book.js (hunk 1 above)
Comment (ese-varo) [reply src/routes/book.js]: just if there are more than one middleware to process the request, in this case it is not necessary to use it

File: server.js (hunk 3 above)
Comment (ese-varo) [reply server.js]: yeap, I'll do it

**Review by GregVaz — APPROVED** [review]
Great work :rocket:

**Pull request merged**